aeppl/transforms.py

```diff
@@ -240,8 +320,24 @@ def log_jac_det(self, value, *inputs):
 def create_default_transformed_rv_op(
     rv_op: Op,
     transform: RVTransform,
+    *,
+    default: bool = True,
```

Comment:
I feel the default should be there option with least side effects (and these dispatch side effects are quite significant imho). Even though it definitely makes things more verbose for what is the most common use.